frontend/src/components/FerryProvider.tsx

```diff
@@ -93,21 +95,25 @@ const addToCache = (season: Season): Promise<void> => {
       [season]: true,
     };
 
-    return axios.get(`/api/static/catalogs/${season}.json`).then((res) => {
-      // Convert season list into a crn lookup table.
-      const data = res.data as Listing[];
-      const info = new Map<Crn, Listing>();
-      for (const raw_listing of data) {
-        const listing = preprocess_courses(raw_listing);
-        info.set(listing.crn, listing);
-      }
-
-      // Save in global cache. Here we force the creation of a new object.
-      courseData = {
-        ...courseData,
-        [season]: info,
-      };
-    });
+    return axios
+      .get(`${API_ENDPOINT}/api/static/catalogs/${season}.json`, {
+        withCredentials: true,
+      })
+      .then((res) => {
+        // Convert season list into a crn lookup table.
+        const data = res.data as Listing[];
+        const info = new Map<Crn, Listing>();
+        for (const raw_listing of data) {
+          const listing = preprocess_courses(raw_listing);
```

Comment:
I love snake case as much as the next python programmer, but why break the JS camel case standard only here?

Reply:
A leftover from our existing script, but it's worth changing